### ISSUE TITLE
Wireframes for Home page

### ISSUE THREAD

**Issue by acabunoc** [mozillascience/site#87]
We're eager to see UI and UX work on our home page. :house: 

See #86 for complementary ticket for content

cc @my-le @kristinashu

**Comment by stephwright** [comment]
My thoughts are inspired by the content issue #86 but my thoughts are more related to UI/UX so putting here.

1) I'd like us to be thoughtful about how the home page appears to someone who doesn't know what MoFo and Sci Lab are about.  That's why I like the verb tabs because it tells folks what they can do.  As much as I love alliteration, not a fan of the three Cs because the words don't seem like they are selected for ease of understanding of a newbie (aren't you contributing if you're collaborating? and what exactly does convening mean to a newbie?).
2) Would LOVE for access to the main content to be above the fold as in @auremoser 's examples 1-5 in #86 .  I understand designing for mobile and how mobile users are used to scrolling.  Can we design for mobile and desktop users though?  I've found myself leaving more than one website because I get tired of scrolling to find what I'm looking for down a long page.

**Comment by my-le** [comment]
To help me create zonings and wireframes for the Home Page, I started to list Design Principles.
Based on “mozillascience.org homepage content” etherpad: https://public.etherpad-mozilla.org/p/sciencelab-homepage-content
Here is a work document in three parts:
1) Visiting Home Page one MUST find, learn, understand that: <---> 2) MSL Special Features are <---> 3) Design Principles WiP
Feel free to comment it on Red Pen so I can improve it: https://redpen.io/p/kwe0b1b8415da9f6f4

![msl-designprinciples-lyt01](https://cloud.githubusercontent.com/assets/10426601/13233180/ca8571f2-d9b2-11e5-9664-f1fc1585e8e6.jpg)

**Comment by hannahkane** [comment]
@my-le - thanks for this. I added some comments, but I don't know how much they will inform this particular document. They're more like discussion prompts for the entire group.

also, during a smaller group meet this morning to talk more about the homepage, we decided a few of us (@kaythaney and me + @kristinashu) would come up with some potential copy ideas for the homepage. Those should give you more to play around with, though of course you might be coming up with copy ideas as well.

**Comment by my-le** [comment]
@hannahkane perfect! I wish this document coulb be used by everyone as work document, so we can work in parallel.

**Comment by hannahkane** [comment]
When we draft copy, we can put that in a google doc or etherpad so everyone can help edit!

**Comment by my-le** [comment]
Wireframe suggestions start here: https://redpen.io/tk575e9dac8ca7359b

**Comment by kaythaney** [comment]
thanks, both! @hannahkane - i flagged some copy suggestions in the doc for
you further down. happy to jump back in, too.

On Mon, Feb 22, 2016 at 5:14 PM, hannahkane <notifications@github.com>
wrote:

> @my-le <https://github.com/my-le> - thanks for this. I added some
> comments, but I don't know how much they will inform this particular
> document. They're more like discussion prompts for the entire group.
>
> also, during a smaller group meet this morning to talk more about the
> homepage, we decided a few of us (@kaythaney
> <https://github.com/kaythaney> and me + @kristinashu
> <https://github.com/kristinashu>) would come up with some potential copy
> ideas for the homepage. Those should give you more to play around with,
> though of course you might be coming up with copy ideas as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozillascience/site/issues/87#issuecomment-187413034>.
>



-- 
@kaythaney <http://twitter.com/kaythaney/>
@MozillaScience <http://twitter.com/mozillascience>

**Comment by kaythaney** [comment]
thanks, my!

On Thu, Feb 25, 2016 at 11:58 AM, My Le <notifications@github.com> wrote:

> Wireframe suggestions start here: https://redpen.io/tk575e9dac8ca7359b
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozillascience/site/issues/87#issuecomment-188877826>.
>



-- 
@kaythaney <http://twitter.com/kaythaney/>
@MozillaScience <http://twitter.com/mozillascience>

**Comment by hannahkane** [comment]
Rolling this into next heartbeat to give everyone time to comment and for @my-le to iterate and come up with new variations based on feedback from the team.

**Comment by kristinashu** [comment]
I think @my-le has done pretty much as much as she can (https://redpen.io/p/kwe0b1b8415da9f6f4) until we get the analytics, user IA feedback and finalized copy.

**Comment by kristinashu** [comment]
@hannahkane should I close this and open a new one about editing the home page wireframes based on upcoming info?

**Comment by hannahkane** [comment]
Yes, please. Thanks, @kristinashu.

**Comment by kristinashu** [comment]
Closing and opening a new ticket for edits https://github.com/mozillascience/site/issues/95
